lmp/bb-config.sh

```diff
@@ -172,6 +172,13 @@ SOTA_PACKED_CREDENTIALS = ""
 EOFEOF
 fi
 
+if [ -n "${LMP_ROLLBACK_PROTECTION_ENABLE}" ]; then
+	status "Configuring boot firmware rollback protection, LMP_ROLLBACK_PROTECTION_ENABLE = ${LMP_ROLLBACK_PROTECTION_ENABLE}"
+	cat << EOFEOF >> conf/local.conf
+LMP_ROLLBACK_PROTECTION_ENABLE ?= "${LMP_ROLLBACK_PROTECTION_ENABLE}"
+EOFEOF
+fi
+
```

Comment:
maybe you can add a default value on top of the file https://github.com/foundriesio/ci-scripts/pull/316/files#diff-f12b382648c4eb1512e3828fbc501e21e138c7b08fba2f6d74410c22ae84bf4aR24-R26
```
LMP_ROLLBACK_PROTECTION_ENABLE="${LMP_ROLLBACK_PROTECTION_ENABLE-0}"
```
and change it to enable only when "1"
```
if [ "${LMP_ROLLBACK_PROTECTION_ENABLE}" = "1" ]; then
	cat << EOFEOF >> conf/local.conf
LMP_ROLLBACK_PROTECTION_ENABLE = "${LMP_ROLLBACK_PROTECTION_ENABLE}"
EOFEOF
fi
```
otherwise we take the default lmp value
```
meta-lmp-base/conf/distro/lmp.conf:63:LMP_ROLLBACK_PROTECTION_ENABLE ??= "0"
```

Reply:
done, thanks!